src/store/messages/Messages.ts

```diff
@@ -0,0 +1,5 @@
+import MessageCollection from '@/datamodel/MessageCollection';
+
+export default interface MessageState {
+	messages: MessageCollection;
```

Comment:
I thought about that point a lot...but I do not see why are cannot do that like with the keys during the initialization for to major reasons:
-> The idea to provide messages in specific language would only hit us, if the Termbox get the possibility to change the user language(aka primary language), right? In my opinion the Termbox is not supposed to do that and the change should happen somewhere above. Otherwise we would be inconsistent with other components (like Statements) in terms of the shown messages. To implement a update of the show messages should be performed by a refresh of the store or the store (for in terms of the client-side) points only the to getMessages and does not contain directly the message string. In case of server side it does not matter because the server will be pinged only if we refresh.